kit/brain_board/index.md

```diff
@@ -13,21 +13,53 @@ The below will automatically calculate the latest version number, to be used whe
 Brain Board
 ===========
 
-<img src="{{ site.baseurl }}/images/content/kit/brain.png" alt="A photo of a brain board" title="A brain board" class="right" />
-The Odroid U3+ Brain Board provided with your kit is what runs the code you write
-and controls the other peripheral boards.
+<img src="{{ site.baseurl }}/images/content/kit/brain.png" alt="A photo of a brain board" title="A brain board" class="right" style="max-width: 50%"/>
+The Brain Board provided with your kit is what runs the code you write and controls the other peripheral boards. It consists of a [Raspberry Pi 4B](https://www.raspberrypi.com/products/raspberry-pi-4-model-b/) and a Student Robotics KCH HAT.
 
-### Flashing SD card
+The LEDs on the HAT display the current status of the robot and can be used to help debug your robot. There are also 3 RGB (Red, Green and Blue) LEDs that you can control from your code.
 
-In order to get access to the SD card, you will need to open up the case for the brain board. Pull the latch at the end of the case in order to hinge it open.
-The SD card is located on the underside of the board. Carefully open the Brain Board's plastic case, grab the SD card with your fingers and simply pull it out of the slot.
+## Board Diagram
+
+![brain board assembly diagram]({{ site.baseurl }}/images/content/kit/brain-diagram.png "The Brain Board Assembly")
```

Comment:
Suggestion: We should just embed the SVG version here, instead of adding an extra PNG to the repo. Client coverage is very good for this: https://caniuse.com/svg-img

Reply:
The SVG doesn't render properly in Chrome or Firefox. I've already tried this.